test/blackbox-tests/test-cases/pkg/dev-tool-conflict-test.t

```diff
@@ -0,0 +1,256 @@
+Testing the conflicts that could occurs between the dependencies of dune-projects
+and dev-tools dependencies.
+
+The scenario here is, the fake ocamlforamt dev-tool depends on
+printer.1.0 and the project depends on a different version, printer.2.0.
+It shows those 2 does not conflicts and the dev tools dependencies do not leak
```

Comment:
Should the "dev tools" in li.6 also be hyphenated "dev-tools"?

Reply:
"dev-tools" is the correct way.